sale_partner_incoterm/__openerp__.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Change the comments as per the latest OCA convention
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3